tests/utils.py

```diff
@@ -103,3 +103,6 @@
 MODELS_TO_TEST_FOR_SPEECH_RECOGNITION = ["wav2vec2", "whisper"]
 
 MODELS_TO_TEST_FOR_IMAGE_TEXT = ["clip"]
+
+# This will be configured by conftest.py at the start of the pytest session
+OH_TEST_DEVICE = "unknown"
```

Comment:
Upon further contemplation, I think `OH_DEVICE_CONTEXT` is a better name.  The `OH_TEST_DEVICE` carries the potential assumption that this actual, physical, device is being tested.  However, users are allowed to set the device to any value, regardless of physical device matching.  That is, user can run "gaudi1" tests and refs on a "gaudi2" if they wanted to.
Hence, `OH_DEVICE_CONTEXT` carries less assumption about physical device presence.  I will update the PR.